dwave_networkx/generators/working_graph.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2018 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# ================================================================================================
+"""
+Generators for graphs derived from the D-Wave System.
+
+"""
+import networkx as nx
+
+from dwave_networkx import _PY2
+from dwave_networkx.exceptions import DWaveNetworkXException
+
+from dimod.decorators import graph_argument
+
+__all__ = ['shared_working_graph']
+
+# compatibility for python 2/3
+if _PY2:
+    range = xrange
+
+
+import numpy as np
+
+@graph_argument('g1', 'g2')
+def shared_working_graph(g1, g2):
```

Comment:
I think these should be more explicitely named `graph0, graph1`, or (following NetworkX convention) `H, G`

Reply:
Also, this function is more general. I think it is better named `common_subgraph` or similar. There is nothing Chimera or Pegasus-specific about it.